rotkehlchen/api/v1/schemas.py

```diff
@@ -2854,3 +2854,20 @@ def make_nft_filter_query(  # pylint: disable=no-self-use
             'ignore_cache': data['ignore_cache'],
             'filter_query': filter_query,
         }
+
+
+class EventDetailsQuerySchema(Schema):
+    event_identifier = EVMTransactionHashField(required=True)
+    sequence_index = fields.Integer(required=True)
+
```

Comment:
shouldn't the frontend requests multiple for the same transaction? If a tx has for example 3 swaps you have to query this endpoint 3 times. Is this something agreed with the frontend?

Reply:
Yes, the idea of returning the details is to keep as minimum data on frontend as possible. So if there are 3 swaps, users will open/expand them one-by-one, not all 3 at the same time. So to keep minimum info on the frontend they request the details only for a specific event